tools/failures.py

```diff
@@ -31,9 +32,12 @@ def get_raw_data(start_date, end_date):
 
     url = "http://alertmanager.allizom.org/data/seta/?startDate=%s&endDate=%s" % \
           (start_date.strftime("%Y-%m-%d"), end_date.strftime("%Y-%m-%d"))
-
-    response = retry(requests.get, args=(url, ),
-                     kwargs={'headers': headers, 'verify': True})
+    try:
+        response = retry(requests.get, args=(url, ),
+                         kwargs={'headers': headers, 'verify': True})
+    except Exception as e:
+        logger.debug("The request to %s is failed due to %s" % (url, e))
+        return
```

Comment:
can we return nothing here and all code that calls this method will be fine?  should we return a  blank object {} ?